src/Constants.cpp

```diff
@@ -22,9 +22,44 @@
 
 namespace DocLayerConstants {
 
-extern const uint64_t INDEX_KEY_LENGTH_LIMIT = (uint64_t)1e4;
+const uint64_t INDEX_KEY_LENGTH_LIMIT = (uint64_t)1e4;
 
-extern const uint64_t FDB_KEY_LENGTH_LIMIT = (uint64_t)1e4;
-extern const uint64_t FDB_VALUE_LENGTH_LIMIT = (uint64_t)1e5;
+const uint64_t FDB_KEY_LENGTH_LIMIT = (uint64_t)1e4;
+const uint64_t FDB_VALUE_LENGTH_LIMIT = (uint64_t)1e5;
+
+const std::string METADATA = "metadata";
+const std::string VERSION_KEY = "version";
+const std::string INDICES_KEY = "indices";
+
+const std::string SYSTEM_INDEXES = "system.indexes";
+const std::string SYSTEM_NAMESPACES = "system.namespaces";
+
+const char* ID_FIELD = "_id";
+const char* KEY_FIELD = "key";
+const char* BUILD_ID_FIELD = "build id";
+const char* STATUS_FIELD = "status";
+const char* NAME_FIELD = "name";
+const char* UNIQUE_FIELD = "unique";
+const char* BACKGROUND_FIELD = "background";
+const char* METADATA_VERSION_FIELD = "metadata version";
```

Comment:
Would it be nicer to make all of them `std::string`? Whenever you need C string (`const char*`), you can get using `.c_str()`. Also `getField(const char *)` has overload function for `getField(std::string)`. But other functions like `getStringField()` don't have overload functions, it might look bit ugly there. I leave it to your discretion, just want to bring it up.

Reply:
I guess I prefer `char*` since the reason why I used `char*` from the beginning was to avoid calling `.c_str()` everywhere LOL